go/vt/vtgate/executor.go

```diff
@@ -314,6 +317,7 @@ func (e *Executor) StreamExecute(
 		// 5: Log and add statistics
 		logStats.Keyspace = plan.Instructions.GetKeyspaceName()
 		logStats.Table = plan.Instructions.GetTableName()
+		logStats.TablesUsed = plan.TablesUsed
```

Comment:
Is keyspace and table are still kept in logStats for backward compatibility?
If so, we should mark it for deprecation